pkg/router/template/plugin.go

```diff
@@ -143,7 +143,7 @@ func NewTemplatePlugin(cfg TemplatePluginConfig, lookupSvc ServiceLookup) (*Temp
 func (p *TemplatePlugin) HandleEndpoints(eventType watch.EventType, endpoints *kapi.Endpoints) error {
 	key := endpointsKey(endpoints)
 
-	glog.V(4).Infof("Processing %d Endpoints for Name: %v (%v)", len(endpoints.Subsets), endpoints.Name, eventType)
+	glog.V(4).Infof("Processing %d Endpoints for %s/%s (%v)", len(endpoints.Subsets), endpoints.Namespace, endpoints.Name, eventType)
```

Comment:
Nothing wrong with this fix but I was thinking we can use endpointsKey and routeKey for the log message. 
Both the keys will print namespace and the name. endpointsKey will print exactly the same thing `<namespace>/<endpoints-name>` and routeKey will print `<namespace>:<route-name>`  (this makes log message consistent with internal representation)

Reply:
Seems good to me either way.